ui/v2.5/src/index.scss

```diff
@@ -23,16 +23,40 @@ body {
   -moz-osx-font-smoothing: grayscale;
   margin: 0;
   padding: 4rem 0 0 0;
+}
 
-  @media (min-width: 600px) {
-    min-width: 845px;
-  }
+a {
+  color: $primary;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
 }
 
+.input-control, .text-input {
+  color: $text-color;
+  border: 0;
+  box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
+}
+
+.input-control:focus, .text-input:focus {
+  color: $text-color;
+  border: 0;
+  box-shadow: 0 0 0 1px $primary, 0 0 0 1px $primary, 0 0 0 3px rgba(19,124,189,.3), inset 0 0 0 1px rgba(16,22,26,.3), inset 0 1px 1px rgba(16,22,26,.4);
+}
+
+.text-input, .text-input:focus {
+  background-color: $textfield-bg;
+}
+
+.input-control, .input-control:focus {
+  background-color: $secondary;
+}
+
```

Comment:
Would it be possible to apply these styles to all input elements rather than classes? 
`input[type=text], textarea, select {
  ...
}`

Reply:
The default type of a Form.Control is text, but yeah it can be hard to override a lot of the bootstrap styles due to precedence rules. No big deal either way.